src/components/editor/ImportResourceTemplate.jsx

```diff
@@ -139,7 +148,21 @@ class ImportResourceTemplate extends Component {
 
 ImportResourceTemplate.propTypes = {
   children: PropTypes.array,
-  triggerHandleOffsetMenu: PropTypes.func
+  triggerHandleOffsetMenu: PropTypes.func,
+  authenticationState: PropTypes.object
```

Comment:
per `LoginPanel` comment, should just take `currentUser` (and `currentSession`? `authenticationError`?) directly, instead of the containing hash.  see #528.